release-notes/6.0/README.md

```diff
@@ -5,3 +5,4 @@ The following Microsoft.Data.SqlClient 6.0 preview releases have been shipped:
 | Release Date | Version | Notes |
 | :-- | :-- | :--: |
 | 2024-08-27 | 6.0.0-preview1.24240.8 | [release notes](6.0.0-preview1.md) |
+| 2024-10-25 | 6.0.0-preview2 | [release notes](6.0.0-preview2.md) |
```

Comment:
@ErikEJ is this the spot you were mentioning for the index files? Or are there others you know of as well?

Reply:
There is a higher level file, but not sure if that usually includes previews?

Reply:
Got it, updated that file.